datarobot_batch_scoring/batch_scoring.py

```diff
@@ -772,7 +807,8 @@ def run_batch_predictions(base_url, base_headers, user, pwd,
                           out_file, keep_cols, delimiter,
                           dataset, pred_name,
                           timeout, ui, fast_mode, auto_sample,
-                          dry_run, encoding, skip_dialect):
+                          dry_run, encoding, skip_dialect,
+                          max_batch_size=3.5 * 1024 ** 2):
```

Comment:
I think the overall goal is to keep people from trying to send 500MB chunks. I'm not sure we need to be that precisely about batch size.